permissions/plugin-packageversion.yml

```diff
@@ -0,0 +1,6 @@
+---
+name: "packageversion"
+paths:
+- "org/lilicurroad/jenkins/packageversion"
```

Comment:
The groupid is typically a reverse domain name. `lilicurroad.org` seems to be unregistered, so that's an odd choice.